oci/runtime_oci.go

```diff
@@ -35,9 +37,6 @@ const (
 
 	// Command line flag used to specify the run root directory
 	rootFlag = "--root"
-
-	// default run root to be used for OCI runtime if it is not set by handler
-	defaultRunRoot = "/run/runc"
```

Comment:
@haircommander We now can move this to the config package since we removed the possible import cycle with `github.com/cri-o/cri-o/lib` and the config now lives in an own package `github.com/cri-o/cri-o/lib/config`

Reply:
:+1: